exporter/hqmf_template_resolver.go

```diff
@@ -135,3 +138,20 @@ func codeDisplayForQrdaOid(oid string) []models.CodeDisplay {
 	}
 	return []models.CodeDisplay{}
 }
+
+func VsToQrdaOidPatientCharacteristic(vsOid string) string {
+	switch vsOid {
+	case "2.16.840.1.113883.3.117.1.7.1.402", "2.16.840.1.113883.3.117.1.7.1.403",
+		"2.16.840.1.113883.3.117.1.7.1.287", "2.16.840.1.113883.3.117.1.7.1.307":
+		// Patient Charasteristic Gestational Age
+		return "2.16.840.1.113883.10.20.24.3.101"
+	case "2.16.840.1.113883.3.526.3.1279", "2.16.840.1.113883.3.526.3.1189", "2.16.840.1.113883.3.526.3.1170", "2.16.840.1.113883.3.600.2390":
+		// Patient Characteristic Observation Assertion template for
+		// Patient Characteristic: ECOG Performance Status-Poor
+		// Patient Characteristic Tobacco User/Non-User
+		return "2.16.840.1.113883.10.20.24.3.103"
+	default:
```

Comment:
Can probably simplify this, as the second `case` statement and the `default` statement both return the same OID.